lexer/src/main.c

```diff
@@ -136,23 +150,31 @@ int main(int argc, char** argv) {
     stat(filename,&file_meta);
 
 	if( access(tkn_filename,F_OK) == -1){
```

Comment:
Very inconsistent indentation. Hard to read.

Reply:
github :(